doc/guides/zephyr_cmake_package.rst

```diff
@@ -0,0 +1,430 @@
+.. _cmake_pkg:
+
+Zephyr CMake Package
+####################
+
+The Zephyr `CMake package`_ is a convenient way to create a Zephyr-based application.
```

Comment:
Isn't that what i'm doing in the following sentence, if you read it all:

```
The Zephyr CMake package is a convenient way to create a Zephyr-based application.

The Zephyr CMake package ensures that CMake can automatically select a Zephyr 
to use for building the application, whether it is a Zephyr repository application, 
Zephyr workspace application, or a Zephyr freestanding application.
```

Reply:
I was thinking about this, and to me, the second sentence doesn't narrow the scope of the document, because it doesn't say whether that is the only thing the Zephyr CMake package does, or if it's just one of many.